adjust.py

```diff
@@ -81,10 +87,11 @@ def run(self):
 
         # Parse input
         try:
-            self.debug("Reading stdin")
-            self.input_data = json.loads(sys.stdin.read())
+            # self.debug("Reading stdin")
+            input_data = json.loads(sys.stdin.read())
+            self.input_data = input_data # LEGACY mode, remove when drivers are updated to use arg
```

Comment:
Shouldn't this also come with a change in the prototype for the base adjust() method, i.e. `adjust(input=None)`?